scan-engine/src/main/java/de/tillhub/scanengine/theme/Theme.kt

```diff
@@ -13,7 +13,7 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.tooling.preview.Preview
 
 @Composable
-internal fun ScanEngineTheme(
+internal fun AppTheme(
```

Comment:
Why did you change the name from ScanEngineTheme -> AppTheme ?

Reply:
Just to keep consistent with how we did in other apps but I reverted it as its not necessary.